docs/modules/ROOT/pages/installation.adoc

```diff
@@ -25,9 +25,6 @@ in your `Gemfile`:
 gem 'rubocop', '~> 1.28', require: false
 ----
 
-NOTE: You can check out our progress on the road to version 1.0 https://github.com/rubocop/rubocop/milestone/4[here].
-You can also help us get there faster! :-)
```

Comment:
Good catch! Can you replace it with the following instead of deleting it?

```
See xref:versioning.adoc[our versioning policy] for further details.
```

cf: https://github.com/rubocop/rubocop/commit/0bb3303d662c0d456f0d81033ac196bb6be5e309